### PR TITLE
node: add German translation

### PR DESCRIPTION
**Pull request by schwarmco**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by pixelcmtd** [comment]
You can put multiple translations into the same PR. When you're doing a few right after each other, please don't create so many individual PRs.

File: pages.de/common/node.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# node
+
+> Server-seitige JavaScript Plattform (Node.js).
+> Weitere Informationen: <https://nodejs.org>.
+
+- Führe eine JavaScript Datei aus:
+
+`node {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.de/common/node.md:8]: Please translate the arguments for get this easier for non-English speakers.
```suggestion
`node {{pfad/zu/datei}}`
```

File: pages.de/common/node.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`node -e "{{code}}"`
+
+- Evaluierung und Ausgabe des Ergebnisses. Nützlich, um die Versionen der Abhängigkeiten von Node zu sehen:
+
+`node -p "{{process.versions}}"`
+
+- Aktiviere Inspector und pausiere die Ausführung bis sich ein Debugger verbindet sobald der Quellcode vollständig geparsed ist:
+
+`node --no-lazy --inspect-brk {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.de/common/node.md:24]:
```suggestion
`node --no-lazy --inspect-brk {{pfad/zu/datei}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**